### ISSUE TITLE
--log options to log only error entries

### ISSUE THREAD

**Issue by xinxilas** [NebuTech/NBMiner#199]
Create a option to log only crash, cuda error, or error messages to be easier to find which card crashed in entire a rig

It will be useful!

**Comment by nebutech-admin** [comment]
@xinxilas You can search `ERROR` or `WARN` in log file

**Comment by xinxilas** [comment]
Yes but if possible to log only errors would be nice, so the file size will not get bigger and bigger, and its much easier to open and read 

Just a suggestison, if it is not so important, no problem
